.github/workflows/default.yml

```diff
@@ -0,0 +1,32 @@
+name: default
+
+on: push
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go 1.11.4
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.11.4"
+      - name: fmt
+        run: test -z $(gofmt -l .)
```

Comment:
golangci-lint can also run this checker. [Just need to enable it explicitly](https://golangci-lint.run/usage/linters#disabled-by-default-linters--e--enable)